.ruby-version

```diff
@@ -1 +1 @@
-2.6.4
+2.6.5
```

Comment:
Does this imply that we should match this version in our projects too? Perhaps this is just the runtime for this project and the others are ok to be on `2.6.3`? AKA here is [site](https://github.com/optimumenergyco/tesla-site/blob/master/.ruby-version).

Reply:
Nope. Gems tend to be very flexible in the versions of Ruby that they support. Usually, authors of big gems try to stick to core language features. In our case, all of the Ruby code is in the Rubocop gem itself, so it shouldn't matter.